modules/integration-test/src/util/store.ts

```diff
@@ -0,0 +1,54 @@
+import { CFCoreTypes, StorePair } from "@connext/types";
+
+export class MemoryStoreService implements CFCoreTypes.IStoreService {
```

Comment:
Looks like this was copy/pasted from `modules/cf-core/test/services/memory-store-service.ts`, we should create a `@connext/store` package (same flavor as `@connext/messaging`) that includes one implementation of this in-memory-store so we don't have to maintain it in two places.